<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlbumWidget</class>
 <widget class="QWidget" name="AlbumWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="boutonSupprimer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>140</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boutonHaut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>170</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="listeChanson">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>331</width>
     <height>231</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <column>
    <property name="text">
     <string>Titre</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Chemin</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="boutonAjouter">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>110</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="boutonBas">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>200</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSubmit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>331</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="labelTitre">
      <property name="text">
       <string>Titre:</string>
      </property>
      <property name="buddy">
       <cstring>lineEditTitre</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditTitre"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelArtiste">
      <property name="text">
       <string>Artiste:</string>
      </property>
      <property name="buddy">
       <cstring>selectArtiste</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="selectArtiste"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelDate">
      <property name="text">
       <string>Publication:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="selectDate">
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::PlusMinus</enum>
      </property>
      <property name="minimum">
       <number>1000</number>
      </property>
      <property name="maximum">
       <number>3000</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="buttonEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>230</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
